### DISCUSSION TITLE
EvoTreesLSS?

### DISCUSSION THREAD

**Discussion by ParadaCarleton** [StatMixedML/XGBoostLSS#51] [Ideas]
Hi! First, I just wanted to say thank you so much for XGBoostLSS and LightGBMLSS, they're amazing packages and super useful :)

I wanted to ask, have you considered adding support for an [EvoTrees](https://github.com/Evovest/EvoTrees.jl) backend? I think this would be incredibly helpful; EvoTrees.jl is the main package I use for regression trees. Thank you!

**Comment by StatMixedML** [comment]
@ParadaCarleton Thanks for your interest in the project and your kind words!

I came across EvoTrees some time ago and found it interesting. Having a Julia version of GBMLSS is something that has been on my ToDo for quite some time now. Is there an analogue to PyTorch-Distributions / Pyro in Julia that allows for calculating grads and hess via automatoc differentiation?

**Reply by ParadaCarleton** [reply]
Yes! Distributions.jl and DistributionsAD.jl would be the relevant packages.
